modules/rosa-hcp-sts-creating-a-cluster-ext-auth-cluster-cli.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-cluster-cli_{context}"]
+= Creating a {hcp-title} cluster that uses external authentication providers
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+Use the ROSA CLI's `--external-auth-providers-enabled` flag to create your cluster that uses an external authentication service.
+
+.Procedure
+
+. Use one of the following commands to create your ROSA with HCP cluster:
++
+* If you did not set environmental variables, run the following command:
++
+[NOTE]
+====
+When creating a {hcp-title} cluster, the default machine Classless Inter-Domain Routing (CIDR) is `10.0.0.0/16`. If this does not correspond to the CIDR range for your VPC subnets, add `--machine-cidr <address_block>` to the following commands.
+====
++
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=<cluster_name> \
```

Comment:
I think this section is reusing the main "how to create a cluster" in 

https://file.rdu.redhat.com/eponvell/OSDOCS-9614_External-OIDC-Auth-HCP/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-sts-creating-a-cluster-cli_rosa-hcp-sts-creating-a-cluster-quickly

I'd highlight the parameters passed, and at least re-inforce the `--external-auth-providers-enable` flag.